### ISSUE TITLE
Add support for CSS Variables autocomplete

### ISSUE THREAD

**Issue by widersky** [zed-industries/community#2380]
### Check for existing issues

- [X] Completed

### Describe the feature

As I wrote in the title. CSS variables have been around for a long time and it would be great to have support right in the box. VSCode has an [extension](https://github.com/willofindie/vscode-cssvar) that provides the right functionality and works quite well.

**Comment by CurbaiCode** [comment]
I don't mean to discredit this issue, but I believe that CSS variable autocomplete already exists in Zed. For me, with or without typing `--`, suggestions pop up.

![Screen Shot 1](https://github.com/zed-industries/community/assets/47488521/1d96d8c3-a32f-4b97-b3f3-a5aa22dfa11f)
![Screen Shot 2](https://github.com/zed-industries/community/assets/47488521/f7f60468-7d7e-4606-bad1-4bcd19bca59d)

---
Zed: v0.118.1 (Zed)
OS: macOS 12.6.6
Memory: 16 GiB
Architecture: x86_64

**Comment by widersky** [comment]
@CurbaiCode Of course, it works if you refer to a variable in the same file, but if you do it in another file, it no longer works.

**Comment by CurbaiCode** [comment]
Ohh, OK. Thank you for the clarification.
